### PR TITLE
Update the README and refactor the layout a bit

### PR DESCRIPTION
**Pull request by palnabarun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,128 +1,130 @@
...
+
+- [Github Issues](https://github.com/pythonindia/inpycon2020-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fcode-of-conduct)
+- Head: Sayan Chowdhury
+- Members
+  * Saptak Sengupta
+  * Neha Kandpal
+
 ## Timeline
 ### TLDR;
 * Sponsership going live on 31st July, 2020.
```
Comment (anistark) [inline README.md]: Sponsership -> Sponsorship

**Comment by anistark** [comment]
LGTM. Let's merge this.

**Review by sayanchowdhury — APPROVED** [review]
Looks good, just to attend to @anistark comment.

File: README.md (hunk 1 above)
Comment (palnabarun) [reply README.md]: Ah! This was there earlier. Let me fix it.

**Comment by palnabarun** [comment]
Thanks for the review @anistark @sayanchowdhury :)

**Pull request merged**